cli/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "zip-cli"
+version = "0.0.0"
+authors = [
+    "Danny McClanahan <dmcC2@hypnicjerk.ai>",
+]
+license = "MIT"
+repository = "https://github.com/zip-rs/zip2.git"
+keywords = ["zip", "archive", "compression", "cli"]
+categories = ["command-line-utilities", "compression", "filesystem", "development-tools::build-utils"]
+rust-version = "1.73.0"
+description = """
+Binary for creation and manipulation of zip files.
+"""
+edition = "2021"
+
+[[bin]]
+name = "zip-cli"
+
```

Comment:
Can we split this into 2 `bin` targets, a "lite" one (no features except `deflate-flate2` and one of the `flate2` backends, opt-level = "z") and a "full" one (all compression algorithms, plus `aes-crypto` and either `chrono` or `time`, with the usual optimization for speed given abundant memory)?